client/src/components/PageExplorer/PageExplorer.scss

```diff
@@ -83,7 +74,7 @@ $explorer-header-horizontal-padding: 10px;
   white-space: nowrap;
 
   .icon {
-    color: $c-page-explorer-secondary;
+    @apply w-text-white/50;
```

Comment:
With this change, we’d be using different icon colors between the header and the individual items inside the explorer. Is that intentional?

Reply:
This is intentional, but I'll do another pass and make sure they all match this colour, great catch!